.eslintrc.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
 				allowElseIf: true
 			}
 		],
-		'import/extensions': 'off'
+		'import/extensions': 'off',
+		'no-console': 'off'
```

Comment:
we should discourage using console.log outside of development, let's keep it as a warning
```suggestion
		'no-console': 'warn'
```

Reply:
This `.eslintrc.js` config file only applies to the dev process of this package itself, not the consuming packages.

We use `console.log` to print updates to the person who's using our npx script.

Let me know if i can keep it `off`, we can put it to `warn` for this project if you want.

Reply:
right, I missed it was this project's config